packer/setup.rb

```diff
@@ -187,12 +207,12 @@ def clone_or_update_ruby_by_json(h, work_dir)
   end
 end
 
-Dir.chdir("rails_ruby_bench") do
+Dir.chdir(RAILS_BENCH_DIR) do
   csystem "bash -l -c \"RAILS_ENV=profile ruby seed_db_data.rb\"", "Couldn't seed the database with profiling sample data!"
```

Comment:
I see, I made it enabled when building rubies with RVM, but not by default when installing locally.